package.json

```diff
@@ -17,14 +17,15 @@
     "@dhis2/d2-ui-core": "^1.3.5",
     "@dhis2/d2-ui-forms": "^5.1.0",
     "@dhis2/d2-ui-group-editor": "^5.1.1",
-    "@dhis2/d2-ui-header-bar": "5.1.0",
     "@dhis2/d2-ui-org-unit-dialog": "^5.1.0",
     "@dhis2/d2-ui-org-unit-select": "^5.1.0",
+    "@dhis2/ui": "https://github.com/d2-ci/ui",
     "@material-ui/core": "^3.9.1",
     "@material-ui/icons": "^3.0.1",
     "babel-eslint": "9.x",
     "d2": "^31",
     "d2-manifest": "^1.0.0",
+    "d2-utilizr": "^0.2.16",
```

Comment:
Is `d2-utilitzr` required by `@dhis2/ui`?

Reply:
No, it's required by the data-table/MultipleDataTable.component.js, just one of the tools: ```import isIterable from "d2-utilizr/lib/isIterable";```
Previously it was installed as a dependency of the old header bar

Reply:
Cool, good catch.